gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/resources/organizations/environments/domains/PasswordPolicyResource.java

```diff
@@ -107,4 +109,31 @@ public void update(
                 .subscribe(response::resume, response::resume);
     }
 
+    @POST
+    @Path("/default")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(summary = "Set default policy",
+            operationId = "setDefaultPolicy",
+            description = "User must have the DOMAIN_SETTINGS[UPDATE] permission on the specified domain " +
+                    "or DOMAIN_SETTINGS[UPDATE] permission on the specified environment " +
+                    "or DOMAIN_SETTINGS[UPDATE] permission on the specified organization")
+    @ApiResponses({
+            @ApiResponse(responseCode = "200", description = "Default policy updated"),
+            @ApiResponse(responseCode = "500", description = "Internal server error")})
+    public void setDefaultPolicy(
+            @PathParam("organizationId") String organizationId,
+            @PathParam("environmentId") String environmentId,
+            @PathParam("domain") String domain,
+            @PathParam("policy") String policy,
+            @Suspended final AsyncResponse response) {
+        final var authenticatedUser = getAuthenticatedUser();
+
+        checkAnyPermission(organizationId, environmentId, domain, Permission.DOMAIN_SETTINGS, Acl.UPDATE)
+                .andThen(domainService.findById(domain)
+                        .switchIfEmpty(Maybe.error(() -> new DomainNotFoundException(domain)))
+                        .flatMapSingle(__ -> passwordPolicyService.setDefaultPasswordPolicy(ReferenceType.DOMAIN, domain, policy, authenticatedUser))
+                        .doOnError(error -> log.error("Update Default Password Policy fails: ", error)))
```

Comment:
```suggestion
                        .doOnError(error -> log.error("Update Default Password Policy fails: {}", error)))
```

Reply:
In {} we put arguments to string. When we want to log message and throwable we use: `void error(String var1, Throwable var2);`